esinstall/README.md

```diff
@@ -97,6 +97,7 @@ interface InstallOptions {
   rollup: {
     plugins?: RollupPlugin[];
     dedupe?: string[];
+    exportConditions?: string[];
```

Comment:
can you move this up out of rollup and into the top-level InstallOptions? 

We'll need to use this ourselves, since esinstall actually does some exports resolution logic as well. For example:

```
      const resolvedResult = resolveWebDependency(installSpecifier, {
        cwd,
        packageLookupFields,
      });
```

We actually already have support for this in `packageLookupFields`, but it's just not export map compliant. I actually wonder if it would be enough to use this same existing property (ex: `['svelte']`) to inform both package.json top-level entrypoints AND valid export map entrypoints. I think we'd end up with something like this then:

```diff
- let foundEntrypoint: any = [...packageLookupFields, 'browser:module', 'module', 'main:esnext']
+ let packageEntrypointLookups: string[] = [...packageLookupFields, 'browser:module', 'module', 'main:esnext']
+ let packageExportLookups: string[] = [...packageLookupFields, 'browser', 'import', 'default']
```

what do you think?

Reply:
Hmm.. could do that although hard to say for sure. I guess people will use them the same way. You don't add electron or svelte unless you are looking to resolve those. Looking at https://nodejs.org/api/packages.html#packages_conditional_exports the stock ones are "import", "require", "node", "default".

Only thing is I think the plugin auto adds the defaults so you only need to add the extra's. I can use packageExportLookups fine enough. It works fine for my usage. Unclear if this affects anything else though. The tests still seem broken for me locally lets see how CI goes.

Reply:
yea that's fair, maybe these really should be two different installOptions: `packageLookupFields` and a new `packageExportLookupFields`. In your case you'll probably want to set both, but you may need to set both to different values.

Reply:
Yeah to keep backward compatibility where export maps aren't supported, then the resolution could be different. Generally exports supersede the other entry form but we account for everyone's environment. I did a single option in the commit last night, but let's make it separate just in case. I can make that change tonight.